### DISCUSSION TITLE
Integrating with autocomplete/search bar dropdown?

### DISCUSSION THREAD

**Discussion by catmanjan** [microsoft-search/pnp-modern-search#2536] [Q&A] [answered]
What are the integration capabilities with regards to the search bar dropdown?

At the moment a user has to type their query in and hit enter or click the magnifier icon before they can see results

How can we show results in the dropdown?

![image](https://user-images.githubusercontent.com/137350/201252674-c6c5c288-e0a5-45f7-9d25-eec45f39a7be.png)

**Comment by wobba** [comment, marked as answer]
See https://microsoft-search.github.io/pnp-modern-search/extensibility/ for the available extensibility options. There are no extensibility options to the out of the box header search box.
